lib/src/revset.rs

```diff
@@ -3516,7 +3516,7 @@ mod tests {
             aliases_map: &aliases_map,
             local_variables: HashMap::new(),
             user_email: "test.user@example.com",
-            date_pattern_context: chrono::Utc::now().fixed_offset().into(),
+            current_time: jiff::Zoned::now(),
```

Comment:
nit: this was UTC before. Since these tests wouldn't depend on the current time nor timezone, I think it's better to use UNIX_EPOCH with UTC.